.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+      continue-on-error: true
+      
+    - name: Upload performance reports
+      uses: actions/upload-artifact@v4
+      with:
+        name: performance-reports
+        path: test_results/performance/
+        retention-days: 30
+        
+    - name: Parse CI summary
+      id: parse_summary
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2002:style:2:19: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead

(shellcheck)